kyaml/openapi/kubernetesapi/v1218/doc.go

```diff
@@ -1,6 +1,6 @@
 // Copyright 2022 The Kubernetes Authors.
 // SPDX-License-Identifier: Apache-2.0
 
-package v1218pb
+package v1218
 
 const OpenAPIProtobuf = "v1218pb"
```

Comment:
why is "Protobuf" / "pb" still in the constant? Also, why is there a constant in `doc.go`?

Reply:
removed this file